source/features/stop-redirecting-in-notifications-bar.tsx

```diff
@@ -0,0 +1,25 @@
+import delegate from 'delegate-it';
+import features from '../libs/features';
+import * as pageDetect from '../libs/page-detect';
+
+function onNotificationActionClick(event: delegate.Event<MouseEvent, HTMLButtonElement>): void {
+	const shouldRedirectToInbox = !event.altKey;
+	const actionForm = event.delegateTarget.closest('form')!;
+	actionForm.toggleAttribute('data-redirect-to-inbox-on-submit', shouldRedirectToInbox);
+}
+
+async function init(): Promise<void> {
+	const actionButtonSelector = '.notifications-v2.notification-shelf .js-notification-action button';
```

Comment:
I think only this is needed:

```suggestion
	const actionButtonSelector = '.notification-shelf .js-notification-action button';
```